tools/StaticAnalysis/Exceptions/Az.MixedReality/SignatureIssues.csv

```diff
@@ -0,0 +1,2 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"c:\workspace\ps-pr\artifacts\Debug\Az.MixedReality\Microsoft.Azure.PowerShell.Cmdlets.MixedReality.dll","Microsoft.Azure.Commands.MixedReality.SpatialAnchorsAccount.NewSpatialAnchorsAccountKey","New-AzSpatialAnchorsAccountKey","1","8510","Cmdlet 'New-AzSpatialAnchorsAccountKey' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
```

Comment:
@XiangyuL-Microsoft for this issue of `New-AzSpatialAnchorsAccountKey` not having a default parameter set, how do you feel about introducing a set that doesn't require `-Primary` or `-Secondary` and by default, regenerates both keys for the user?

Reply:
When working on the private repo, I have a discussion with @vladimir-Shcherbakov. This is by design. Two keys are provided to make sure even one of them is regenerated, the other can still work. Users are not expected to re-generate both of them simultaneously, So no default parameter set is provided for this Cmdlet to force user explicitly specifying which key to be regenerated.